packages/docs/src/components/ColorSwatchList.tsx

```diff
@@ -65,7 +66,7 @@ const ColorSwatchList = ({ backgroundClass, colorNames, preface, theme }: ColorS
   return (
     <div className="c-swatch-list ds-u-border--1 ds-u-padding--2">
       {colorList.map(({ name, hex }, index) => (
-        <article className="ds-u-margin-bottom--1 c-swatch" key={`${name}-${index}`}>
+        <article className="ds-u-margin-bottom--1 c-swatch" key={uniqueId(`${name}-`)}>
```

Comment:
Sorry, one more thing. What was the rule that spawned this change? It seems unnecessary to me to use `uniqueId` here because this is a key that is used internally by React. I think there's a rule somewhere about not using index, but we have no perfect guarantees that `name` and `hex` will produce unique results...however 🤔, if the combination of `name` and `hex` are not unique, then there is no reason to render a second instance of one...so maybe there's no harm in using that as a key. All that to say, could we change it to
```suggestion
        <article className="ds-u-margin-bottom--1 c-swatch" key={`${name}-${hex}`}>
```

Reply:
Found the rule: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md, which links to some React docs. After reading that, I stand by my suggestion